internal/test/utils.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package test
+
+import (
+	"testing"
+
+	pb "github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/assert"
+
+	proto "storj.io/storj/protos/overlay"
+	"storj.io/storj/storage"
+)
+
+// NewNodeAddressValue provides a convient way to create a storage.Value for testing purposes
+func NewNodeAddressValue(t *testing.T, address string) storage.Value {
```

Comment:
Is there a better place where this could be?

Also it doesn't look like it's being used.

Reply:
I think this is the best place. I think it's helpful to other team members to have a place to put test utilities so they can be leveraged.

It's being used in overlay_test.go

Reply:
`overlay_test.go` uses some other function, not this one.

Reply:
ok fixed

Reply:
Ah, the reasoning for trying to avoid `utils`-like packages is that they often end up acting like sponges, collecting lots and lots of code, which makes the package without a clear purpose.

Reply:
I think that could happen yes but I also think it's really helpful, especially with Go, to have a place that collects common utilities so we can leverage others work without copy/paste

Reply:
The question itself was more of a rhetorical nature. I'm not disagreeing that there are trade-offs. This is the question I ask myself every time I add a thing to utils package. Is there a better place, could I create a better place for it, what's the cost of duplicating this specific code, is there something of value I'm not currently capturing in the package/code structure etc.